modules/visualizing-external-entities-known-limitations.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * operating/visualizing-external-entities.adoc
+:_mod-docs-content-type: CONCEPT
+[id="visualizing-external-entities-known-limitations_{context}"]
+= Known limitations
+
+[role="_abstract"]
+Following are some known limitations of the Visualizing external entities feature:
+
+* When you enable external IP collection for a cluster, Collector in those clusters reports more information to Sensor and to Central. It might create scalability issues if the workload in the cluster communicates with a large number of distinct external peers. Red{nbsp}Hat recommends that you do not enable this feature on clusters with communication patterns involving more than 10,000 distinct external entities.
+* The display of IP information on the *Network Graph* page is experimental and uses a graphical item for each IP entity. Enabling this level of detail when the number of peers is large might result in a cluttered and potentially unresponsive user interface.
```

Comment:
> the number of peers is large

Do we have a definition of how many _large_ is here, as the previous bullet had a suggested number?

Reply:
Not a precise number, but I think more than a couple of dozen would result in clutter. It might take a thousand to make the UI unresponsive.